UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -2196,6 +2272,19 @@ protected virtual ExtTextureTransformExtension GetTextureTransform(TextureInfo d
 			else return null;
 		}
 
+		protected async Task YieldOnTimeoutAndCheckMemory()
+		{
+			if (_options.CheckForLowMemory)
```

Comment:
Nit: 'Check' is kind of vague. Consider renaming to ThrowOnLowMemory to make it clear what the flag does